cmd/virtiofs-placeholder/main_test.go

```diff
@@ -0,0 +1,67 @@
+package virtiofs_placeholder_test
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var placeholderBinary string
+
+func init() {
+	flag.StringVar(&placeholderBinary, "placeholder-binary", "_out/cmd/virtiofs-placeholder", "path to virtiofs placeholder binary")
```

Comment:
Should be part of the test suite setup?

Reply:
That is how other C code is tested, specifically cmd/container-disk-v2alpha, any suggestion?

Reply:
That means the command line option will be redefined before each test, right?, that will fail

Reply:
True